Libplanet.Explorer/GraphTypes/BlocksQuery.cs

```diff
@@ -89,14 +97,22 @@ private IEnumerable<Block<T>> ListBlocks(bool desc, int offset, int? limit)
             else
             {
                 IEnumerable<Block<T>> blocks = _chain.Skip(offset);
-                if (limit is int l)
-                {
-                    blocks = blocks.Take(l);
-                }
 
                 foreach (Block<T> block in blocks)
                 {
+                    if (!empty && block.Transactions.Count() == 0)
```

Comment:
I think we can use [`.Any()`](https://docs.microsoft.com/ko-kr/dotnet/api/system.linq.enumerable.any?view=netstandard-2.0) here.
```suggestion
                    if (!empty && !block.Transactions.Any())
```